packages/p2p/src/components/my-profile/my-profile-form/my-profile-form.scss

```diff
@@ -41,6 +48,15 @@
                 padding-bottom: 7.4rem + 2.4rem;
```

Comment:
Wondering why we need to have this operation here. :)

Reply:
let me edit it 🙏